h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -241,9 +242,63 @@ public AbstractPrediction predict(RowData data) throws PredictException {
    * @throws PredictException
    */
   public AutoEncoderModelPrediction predictAutoEncoder(RowData data) throws PredictException {
-    double[] preds = preamble(ModelCategory.AutoEncoder, data);
-    throw new RuntimeException("Unimplemented " + preds.length);
+    validateModelCategory(ModelCategory.AutoEncoder);
+
+    int size = m.getPredsSize(ModelCategory.AutoEncoder);
+    double[] output = new double[size];
+    double[] rawData = nanArray(m.nfeatures());
+    rawData = fillRawData(data, rawData);
+    output = m.score0(rawData, output);
+
+    AutoEncoderModelPrediction p = new AutoEncoderModelPrediction();
+    p.original = expandRawData(rawData, size);
```

Comment:
Encoded input. We expose this to the user so that he can calculate the reconstruction error.